ext/standard/array.c

```diff
@@ -1029,11 +1029,54 @@ static inline HashTable *get_ht_for_iap(zval *zv, bool separate) {
 	return zobj->handlers->get_properties(zobj);
 }
 
+enum php_array_iter_undef_strategy {
+	IA_MOVE_FORWARD,
+	IA_MOVE_BACKWARD,
+	IA_THROW,
+};
+
+static ZEND_COLD void ia_throw_undef_error(void)
```

Comment:
I actually thought to rename all symbols... :) It's kind of what is done in this file - most of the functions (including static ones) start with php_array ...

Reply:
Right of course...